lib/auth/init.go

```diff
@@ -105,12 +101,11 @@ func Init(cfg InitConfig, seedConfig bool) (*AuthServer, *Identity, error) {
 		return nil, nil, trace.BadParameter("HostUUID: host UUID can not be empty")
 	}
 
-	lockService := local.NewLockService(cfg.Backend)
-	err := lockService.AcquireLock(cfg.DomainName, 60*time.Second)
+	err := cfg.Backend.AcquireLock(cfg.DomainName, 30*time.Second)
 	if err != nil {
 		return nil, nil, err
 	}
-	defer lockService.ReleaseLock(cfg.DomainName)
+	defer cfg.Backend.ReleaseLock(cfg.DomainName)
```

Comment:
I'd prefer not to expose low-level key-value backend as an interface to auth service. That's why I had locking service. You can simply expose it as a part of `Presence` service to save typing.